emperor/support_files/js/sceneplotview3d.js

```diff
@@ -397,7 +397,12 @@ define([
           text = decomp.axesNames[index];
         }
       }
-      text += ' (' + decomp.percExpl[index].toPrecision(4) + ' %)';
+
+      // account for custom axes (their percentage explained will be -1 to
+      // indicate that this attribute is not meaningful).
+      if (decomp.percExpl[index] >= 0) {
```

Comment:
What about N/A or even better the name of the column used? -1 is an value and perhaps will confuse users. Same when using more than 1 axis.